pkg/msgs/msg_encryption.go

```diff
@@ -178,10 +178,21 @@ func PutObjectInEnvelope(ob interface{}, senderID string, recipientID string) (*
 	if err != nil {
 		return nil, err
 	}
-	return PutMessageInEnvelope(bits, senderID, recipientID)
+	msg, ok := ob.(*bridgemodel.NatsMessage)
+	skipEncrpt := false
+	if ok {
+		parsedMsgSubject, _ := ParseSubject(msg.Subject)
```

Comment:
nit: should this be a condition on the error. If no error look at `parsedMsgSubject`, `skipEncrpt` anyway defaults to `false`.